pyopia/instrument/silcam.py

```diff
@@ -23,41 +25,63 @@ def timestamp_from_filename(filename):
     return timestamp
 
 
+@dataclass
+class LoadData:
+    timestamp: Any
+    img: Any
+
+
 class SilCamLoad():
     '''PyOpia pipline-compatible class for loading a single silcam image
 
+    Requires pipeline data: :class:`LoadData`
+
     Parameters
     ----------
     filename : string
         silcam filename (.silc)
 
     Returns
     -------
-    timestamp : timestamp
-        timestamp from timestamp_from_filename()
-    img : np.array
-        raw silcam image
+        data.img : :class:`LoadData`
+        data.timestamp : :class:`LoadData`
     '''
 
     def __init__(self, filename):
         self.filename = filename
 
-    def __call__(self, data):
+    def __call__(self, data: LoadData) -> LoadData:
```

Comment:
This does not use anything from data, so to show this you can do 
`def __call__(self, _ ) -> LoadData:`
The underscore argument implies that this input is not used